frontend/src/app/subject-comments/subject-comments.component.html

```diff
@@ -0,0 +1,29 @@
+<div>
+    <h2>Comments</h2>
+    <p *ngIf="fatalMessage" class="error">{{fatalMessage}}</p>
+    <p *ngIf="message" class="error">{{message}}</p>
+    <div *ngIf="!fatalMessage">
+        <table *ngFor="i in comments">
```

Comment:
ngFor should always use 'let' keyword and if we can rename 'i' to comment.
s0 -> *ngFor="let comment in comments"

Reply:
fixed :icecream: ... oh though it expected "of" instead of "in", I'm assuming that's correct.